src/wallet/scriptpubkeyman.h

```diff
@@ -254,9 +254,9 @@ class ScriptPubKeyMan
 
     /** Prepends the wallet name in logging output to ease debugging in multi-wallet use cases */
     template <typename... Params>
-    void WalletLogPrintf(util::ConstevalFormatString<sizeof...(Params)> wallet_fmt, const Params&... params) const
+    void WalletLogPrintf(LogFmt<Params...> wallet_fmt, const Params&... params) const
```

Comment:
nit: in tinyformat the print variant that appends a `\n` is called [printfln](https://github.com/bitcoin/bitcoin/blob/master/src/tinyformat.h#L1081-L1086) -> please consider a similar naming (I'm not necessarily recommending it, just bringing it to your attention):
```suggestion
    void WalletLogPrintfln(LogFmt<Params...> wallet_fmt, const Params&... params) const
```

If you disagree, just resolve it, no explanation needed.